Palace.xcodeproj/project.pbxproj

```diff
@@ -3943,6 +3955,7 @@
 				CLANG_ENABLE_MODULES = YES;
 				CLANG_WARN_NON_LITERAL_NULL_CONVERSION = YES_ERROR;
 				CODE_SIGN_ENTITLEMENTS = Palace/SimplyE.entitlements;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_IDENTITY = "iPhone Distribution";
```

Comment:
Something went wrong here... Xcode reports "Palace has conflicting provisioning settings." Could you remove `CODE_SIGN_IDENTITY = "iPhone Developer";` line?